official/resnet/keras/keras_imagenet_benchmark.py

```diff
@@ -139,9 +139,8 @@ def benchmark_8_gpu_mlperf_like_tweaked(self):
     FLAGS.dtype = 'fp16'
     FLAGS.enable_eager = True
     FLAGS.enable_xla = True
-    # Tweaks to improve performance.
-    FLAGS.data_delay_prefetch = True
     FLAGS.use_tensor_lr = True
+    FLAGS.tf_gpu_thread_mode = 'gpu_private'
```

Comment:
Was this one just missing by mistake before?

Reply:
I think we previously missed this one; @tfboyd Could you confirm?